### DISCUSSION TITLE
How to disable rows in DataGrid, and change keyboard navigation to skip them ?

### DISCUSSION THREAD

**Discussion by iksi4prs** [AvaloniaUI/Avalonia#13112] [Q&A]
Hi,

Is it possible to mark some rows as disabled, and change keyboard navigation to skip them ?

Thanks for your help.

**Comment by maxkatz6** [comment]
You can handle LoadingRow event to adjust row properties. Or you can create styles, adding binding to the DataGridRow type selector.

**Reply by iksi4prs** [reply]
Hi @maxkatz6 , thanks for your answer.
I was able to disable the rows. but I want that keyboard navigation, eg with up/down arrows will skip the disabled rows.
I didnt find a simple way to do it.
If I add key bindings to up/down arrows, i need to re-implement all keyboard navigation.
Any suggestion for simpler way to do it ?
10x.
